browser/brave_wallet/brave_wallet_tab_helper.cc

```diff
@@ -34,6 +35,40 @@ void BraveWalletTabHelper::ShowBubble() {
   wallet_bubble_manager_delegate_->ShowBubble();
 }
 
+void BraveWalletTabHelper::UserRequestCompleted(const std::string& requestData,
+                                                const std::string& result) {
+  size_t hash = base::FastHash(base::as_bytes(base::make_span(requestData)));
+  DCHECK(request_callbacks_.count(hash));
+  std::move(request_callbacks_[hash]).Run(result);
+  request_callbacks_.erase(hash);
+}
+
+void BraveWalletTabHelper::RequestUserApproval(
+    const std::string& requestData,
+    RequestEthereumChainCallback callback) {
+  std::string requesting_origin;
+  std::vector<std::string> accounts;
+  auto* manager =
+      permissions::PermissionRequestManager::FromWebContents(web_contents_);
+  DCHECK(manager);
```

Comment:
Not needed at all, please remove this.

Reply:
@spylogsster I meant removing those lines getting permission request manager too.

Reply:
sure, done